server/views/formPages/incidentMacros.njk

```diff
@@ -0,0 +1,124 @@
+{% macro inlineConditionalRadiosWithNestedRadios(question) %}
+
+  <div class="govuk-form-group">
+    <fieldset class="govuk-fieldset">
+      <legend class="govuk-fieldset__legend govuk-fieldset__legend">
+        <h1 class="govuk-fieldset__heading">
+          {{question.primaryQuestion}}
+        </h1>
+      </legend>
+      <div class="govuk-!-margin-bottom-6" >
+        <div class="govuk-radios--inline" data-module="radios">
+          <div class="govuk-radios__item">
+            <input class="govuk-radios__input" id="id-{{question.primaryName}}-1" name="{{question.primaryName}}" type="radio" value="{{question.primaryOpt1}}" data-aria-controls="conditional-{{question.primaryName}}">
+            <label class="govuk-label govuk-radios__label" for='id-{{question.primaryName}}-1'>
+              {{question.primaryOpt1 | capitalize }}
+            </label>
+          </div>
+
+          <div class="govuk-radios__item">
+            <input class="govuk-radios__input" id="id-{{question.primaryName}}-2" name="{{question.primaryName}}" type="radio" value="{{question.primaryOpt2}}">
```

Comment:
If we could structure the question data structure in this way:
```
{
	"primaryQuestion": {
		"text": "Was control and restraint used?",
		"name": "restraint",
		"options": [{
			"value": "yes",
			"label": "Yes"
		}, {
			"value": "yes",
			"label": "Yes"
		}]
	},
	"followUpQuestion": {
		"text": "What positions were used?",
		"name": "restraintPositions",
		"options": [{
			"value": "standing",
			"label": "standing"
		}, {
			"value": "supine",
			"label": "On back (supine)"
		}, {
			"value": "prone",
			"label": "Face down (prone)"
		}, {
			"value": "kneeling",
			"label": "Kneeling"
		}]
	}
}
```

Then it would mean we wouldn't have to hard code the number of options and instead allow us to iterate the array.
See:
https://mozilla.github.io/nunjucks/templating.html#for
Also note the additional variables that are made available to the template: 
`loop.index` 
`loop.index0`